pages/api/replicache-pull.ts

```diff
@@ -17,7 +17,7 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
 
   await transact(async (executor) => {
     [entries, lastMutationID, cookie] = await Promise.all([
-      executor("SELECT * FROM Object WHERE Version > :version", {
+      executor("SELECT K, V, Deleted FROM Object WHERE Version > :version", {
```

Comment:
This was actually working by accident before. The third column was the version field, which ended up always being truthy, so we never went down the deleted path (as expected, but accidentally).

Yes, we need tests. It's on the list.